docs/use-cases/observability/clickstack/event_deltas.md

```diff
@@ -0,0 +1,66 @@
+
+Event Deltas are available directly through the **Search** panel in ClickStack when selecting a source of type `Trace`.
+
+From the top-left **Analysis Mode** selector, choose **Event Deltas** (with a `Trace` source selected) to switch from the standard results table, which displays spans as rows.
+
+<Image img={event_deltas_no_selected} alt="Event Deltas not selected" size="lg"/>
+
+This view renders the distribution of spans over time, showing how latency varies alongside volume. The vertical axis represents latency, while the coloring indicates the density of traces at a given point-darker yellow areas correspond to a higher concentration of traces. With this visualization, users can quickly see how spans are distributed across both latency and count, making it easier to identify shifts or anomalies in performance.
+
+<Image img={event_deltas_highlighted} alt="Event Deltas highlighted" size="lg"/>
+
+Users can then select an area of the visualization - ideally one with higher duration spans and sufficient density, followed by **Filter by Selection**. This designates the "outliers" for analysis. Event Deltas will then identify the columns and key values most associated with those spans in this outlier subset compared to the rest of the dataset. By focusing on regions with meaningful outliers, ClickStack highlights the unique values that distinguish this subset from the overall corpus, surfacing the attributes most responsible for the observed performance differences.
+
+<Image img={event_deltas_selected} alt="Event Deltas selected" size="lg"/>
+
+For each column, ClickStack identifies values that are heavily biased toward the selected outlier subset. In other words, when a value appears in a column, if it occurs predominantly within the outliers rather than the overall dataset (the inliers), it is highlighted as significant. Columns with the strongest bias are listed first, surfacing the attributes most strongly associated with anomalous spans and distinguishing them from baseline behavior.
+
+<Image img={event_deltas_outliers} alt="Event Deltas outliers" size="lg"/>
+
+Consider the example above where the `SpanAttributes.app.payment.card_type` column has been surfaced. Here, the Event Deltas analysis shows that `29%` of the inliers use MasterCard, with `0%` among the outliers, while `100%` of the outliers use Visa, compared to `71%` of the inliers. This suggests that the Visa card type is strongly associated with the anomalous, higher-latency traces, whereas MasterCard appears only in the normal subset.
+
+<Image img={event_deltas_issue} alt="Event Deltas issue" size="lg"/>
+
+Conversely, values exclusively associated with inliers can also be interesting. In the example above, the error `Visa Cash Full` appears exclusively in the inliers and is completely absent from the outlier spans, where this occurs are always less than approximately 50 milliseconds, suggesting this error is associated with low latencies.
```

Comment:
```suggestion
Conversely, values exclusively associated with inliers can also be interesting. In the example above, the error `Visa Cash Full` appears exclusively in the inliers and is completely absent from the outlier spans. Where this occurs, latency is always less than approximately 50 milliseconds, suggesting this error is associated with low latencies.
```

@gingerwizard, original sentence was difficult to read. I believe this is what you wanted to say, but please check.